lib/semian/mysql2.rb

```diff
@@ -62,7 +62,15 @@ def connect(*args)
     end
 
     def semian_resource
-      @semian_resource ||= ::Semian.retrieve_or_register(semian_identifier, **semian_options)
+      @semian_resource ||= case query_options[:semian]
+      when false
+        UnprotectedResource.new(semian_identifier)
```

Comment:
Also do you have any ideas on how we don't have to implement this in every driver? That kind of sucks. I don't have any off the bat though.